openshift/release/release-notes.md

```diff
@@ -0,0 +1,48 @@
+# Tektoncd cli TP-1.3
+
+# Tekton Cli v0.15.0
+
+## Features :sparkles:
+* Basic execution plugin module https://github.com/tektoncd/cli/pull/1178
+* Add --no-headers flag to tkn condition/triggerbinding/eventlistenner list command https://github.com/tektoncd/cli/pull/1259,
+  https://github.com/tektoncd/cli/pull/1254 and https://github.com/tektoncd/cli/pull/1258
+* Allow --prefix-name and --timeout to be override when using --last or --use options https://github.com/tektoncd/cli/pull/1242
+
+## Fixes :bug:
+* Hoist an ActivityTimeout option https://github.com/tektoncd/cli/pull/1265
+* Ignore --follow flag when TaskRun or PipelineRun is done https://github.com/tektoncd/cli/pull/1250
+
+## Deprecation :broom:
+* remove --check shorthand and add global flags to version cmd https://github.com/tektoncd/cli/pull/1269
+
+# Tekton Cli v0.14.0
+
+## Features :sparkles:
+* Update OSX instruction to install from brew :beers: https://github.com/tektoncd/cli/pull/1196
+* Add EventListener Logs Command https://github.com/tektoncd/cli/pull/1192
+* Bump Knative/K8s dependencies https://github.com/tektoncd/cli/pull/1198
```

Comment:
i think we can remove `Bump Knative/K8s dependencies https://github.com/tektoncd/cli/pull/1198`